src/htmx.js

```diff
@@ -1066,6 +1066,10 @@ return (function () {
                     processSSESource(elt, value[1]);
                 }
 
+                if ((value[0] === "forward")) {
```

Comment:
I like "listen" rather than "forward" here: 

```html 
<div hx-sse="listen"> ... </div> 
```

Reply:
I don't have a strong opinion on the name, so I defer to you on this. I've made the change as requested.